spec/rubocop/cop/rspec/scattered_let_spec.rb

```diff
@@ -12,6 +12,76 @@
         ^^^^^^^^^^^^^ Group all let/let! blocks in the example group together.
       end
     RUBY
+
+    expect_correction(<<-RUBY)
+      RSpec.describe User do
+        let(:a) { a }
+        let(:b) { b }
+        subject { User }
```

Comment:
I'm not sure why we use `subject` in examples here, it should be before `let`s. Maybe use a one-line example instead?